lib/interferon.rb

```diff
@@ -169,7 +212,7 @@ def update_alerts_on_destination(dest, hosts, alerts, groups)
       unless @request_shutdown
         # run time summary
         run_time = Time.new.to_f - start_time
-        statsd.histogram('destinations.run_time', run_time, :tags => ["destination:#{dest.class.name}"])
+        statsd.histogram('destinations.run_time', run_time, :tags => ["destination:#{dest.class.name}", "dry_run:false"])
```

Comment:
I personally would prefer the use of an entirely different key for the dry run (e.g. `destinations.run_time.dry_run` or something) as to not confuse any downstream consumers.